source/faq.rst

```diff
@@ -5,23 +5,54 @@ FAQ
 Why don't I have access to browse my Pod?
 =========================================
 
-You must log into your Solid Pod and add PodBrowser as a trusted
-application with ``Read``, ``Write``, ``Append``, and ``Control``
-access to manage your Solid Pod.
+To manage your Pod, PodBrowser must be added as 
+a trusted application with ``Read``, ``Write``, ``Append``, and ``Control``
+access. 
 
-#. Log into your Solid Pod.
+To adjust PodBrowser's access:
+
+#. In a web browser, enter the root URL to you Pod (e.g., https://docs-example.inrupt.net). 
+
+#. The `Solid Data Browser`_ should be opened.
+
+#. Click the :guilabel:`Log In` button to log into your Pod.
 
 #. Go to the :guilabel:`Preferences` tab.
 
-#. In the :guilabel:`Manage your trusted applications` section, add
-   PodBrowser to the table:
+#. In the :guilabel:`Manage your trusted applications` section:
+
+   * If PodBrowser is not listed as a trusted application:
+  
+     #. At the bottom of the table enter:
+
+       .. list-table::
+
+         * - Application URL
+
+           - Access modes
+
+         * - ``https://podbrowser.inrupt.com``
+
+           - ``Read``, ``Write``, ``Append``, and ``Control``
```

Comment:
Oh @kevin-inrupt  - just noticed the .. doesn't line up with A in "At" which is why it's not rendering quite right.

.. should align with the A in At
* - should line up with the l in list-table

```suggestion
        .. list-table::

           * - Application URL

             - Access modes

           * - ``https://podbrowser.inrupt.com``

             - ``Read``, ``Write``, ``Append``, and ``Control``

```